elections/2023/governance-committee-election.md

```diff
@@ -0,0 +1,108 @@
+# OpenTelemetry 2023 Governance Committee election
+
+Please see **[the list of candidates running in the election here](./governance-committee-candidates.md)**.
+
+This document lays out the 2023 election process per the OpenTelemetry governance committee [charter document](../../governance-charter.md#establishment-of-a-governance-committee) requirements.
+
+This election should fill four seats.
+
+Election schedule:
+
+* 13 October 2023 - final deadline to submit nominations
+* 16 October 2023 - official nominees list published
+* 23 October 2023 to 25 October 2023 - voting takes place
+* 27 October 2023 - results announced
+
+We highly encourage participation in this election cycle to ensure that the community is well-represented by the Governance Committee.
+
+# TL;DR
+
+* If you've been nominated or are willing to nominate yourself: check the [charter document](../../governance-charter.md) and confirm you are ready for the commitment. Make sure to provide all necessary information before 13 October 2023 00:00 UTC
```

Comment:
When I read that the deadline is 13 October, my mind registers that I can still submit my application on that date. So, the deadline would be 13 October 2023 23:59 UTC instead?

```suggestion
* If you've been nominated or are willing to nominate yourself: check the [charter document](../../governance-charter.md) and confirm you are ready for the commitment. Make sure to provide all necessary information before 13 October 2023 23:59 UTC
```